src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -504,8 +510,8 @@ def generate_PCR_3_default_config(self, filename):
 
     # SAS UUT claimed ppa boundary generated in PCR.1 test.
     try:
-      with open(sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file:
-        user_claimed_ppa_contour = json.load(claimed_ppa_file)
+      with sas_uut_claimed_ppa_boundary_file_path, 'r') as claimed_ppa_file
+        user_claimed_ppa_contour = json_load(claimed_ppa_file)
```

Comment:
Fixed -- all these json_load  typo are related to a last minute cleaning change that was not reflecting in our internal code base. This has been fixed, and all those change are now reflected there, so that the impedance match is decreased between the 2 versions (google and winnforum versions).